LSP-copilot.sublime-settings

```diff
@@ -12,6 +12,7 @@
 	"initializationOptions": {},
 	"settings": {
 		"auto_ask_completions": true,
+		"hook_to_auto_complete_command": false,
```

Comment:
I guess this is defaulted to `true`, as specified in `sublime-package.json`.

Reply:
Hm, there is a typo in `sublime-package.json` as my idea was to make it false by default


But I am happy to make it `true`

Reply:
Since it's configurable, I think the default is not really important. Just to sync two side either way.

Reply:
Made it `false`